utils/installer/install.sh

```diff
@@ -1,272 +1,280 @@
-#!/bin/sh
-#Set Variable to master is not set differently
-LVBRANCH="${LVBRANCH:-master}"
-USER_BIN_DIR="/usr/local/bin"
-set -o nounset # error when referencing undefined variable
-set -o errexit # exit when command fails
-
-installnodemac() {
-  brew install lua
-  brew install node
-  brew install yarn
-}
+#!/usr/bin/env bash
+set -eo pipefail
 
-installnodeubuntu() {
-  sudo apt install nodejs
-  sudo apt install npm
-}
+#Set branch to master unless specified by the user
+declare -r LVBRANCH="${LVBRANCH:-master}"
```

Comment:
`declare -r LVBRANCH=${LVBRANCH:-"master"}`

Reply:
That's not necessary. I needed to change it to rolling anyway.

Reply:
I'd like that you quote the inner most variable instead of outer so that is also matches other lines